libmpv/qthelper.hpp

```diff
@@ -97,8 +97,8 @@ static inline QVariant node_to_variant(const mpv_node *node)
 }
 
 struct node_builder {
-    node_builder(const QVariant& v) {
-        set(&node_, v);
+    node_builder(const QVariant& v, bool force_list = false) {
```

Comment:
Can you do it without this parameter? I don't really understand what's its purpose is supposed to be and it touches code that has nothing to do with the PR's title.